lib/server/appTokenUtil.js

```diff
@@ -19,6 +19,16 @@ function getAppTokensBasedOnQuery(query) {
   });
 
   let vet = GlobalVets.findOne({appIdentifier: appTokens[0].appName});
+
+  if (!vet && desktopAppId) {
+    vet = GlobalVets.findOne({_id: desktopAppId});
+  }
+
+  if (!vet) {
+    console.log(`Couldn't find vet for id ${desktopAppId} or app identifier ${appTokens[0].appName}`);
+    return false;
+  }
+
```

Comment:
This method should return an array. Also, I think this logic is wrong. After we fetch `appTokens` in line 9 we should check if `desktopAppId` exists. If it does, we should return `appTokens`. If not, we should do  our existing logic.

Reply:
Actually, we need to filter `appTokens` with vetId.

Reply:
@mslobodan this method was/is primarily used to remove disabled users, in order to prevent them from sending notification, that's why `vet` is being fetched.`desktopAppId` is used because the appIdentifier is representing `vet2petdashboard` project, so vet can't be fetched through it. 

We could filter out tokens by vetId, but they are already filtered out in `sendDesktopNotification`